README.md

```diff
@@ -45,6 +45,10 @@ If compared to saving websites with `wget -mpk`, this tool embeds all assets as
  - `-s`: Silent mode
  - `-u`: Specify custom User-Agent
 
+## HTTPS and HTTP proxies
+
+Please set `$https_proxy` and `$http_proxy` environment variables.
```

Comment:
I think since the `$` is used for getting and not setting variables in most environments, it may be better to refer to them as just `https_proxy` and `http_proxy`, perhaps even upper-case them since that's the style I see used in most GUI programs and code snippets on the Internet, but casing is not as relevant.

Reply:
ok, I'll modify them. And environment variables are case-sensitive. `$HTTP_PROXY` is different from `$http_proxy`. It seems that lower cases are more common. For example, documentation of Arch Linux uses lowercases: https://wiki.archlinux.org/index.php/Proxy_server#Environment_variables

Reply:
That's wild! Thanks for the info and updating the readme.